src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+public class Ball {
+    private int number;
+    private String color;
+
+    public Ball(String color, int number) {
+        this.color = color;
+        this.number = number;
+    }
+
+    public String toString(Ball ball) {
+        return "Ball number " + ball.number + " has " + ball.color + " color.";
+    }

```

Comment:
You need to override this method
```suggestion

    @Override
    public String toString() {
        return "Ball number " + number + " has " + color + " color.";
    }
```